atomicfu/src/commonTest/kotlin/kotlinx/atomicfu/test/AtomicArrayTest.kt

```diff
@@ -156,4 +156,20 @@ class AtomicArrayWithMethod {
         val result = refArray[index].compareAndSet(null, data)
         if (!result) error("Double set")
     }
+}
+
+class IndexArrayElementGetter {
+    val clazz = AtomicArrayClass()
+    fun intGet(index: Int) = { clazz.longArr[index].value }
+
+    @Suppress("NOTHING_TO_INLINE")
+    inline fun refGet(index1: Int): Any? = clazz.refArr[index1].value
+
+    @Test
+    fun testIndexArrayElementGetting() {
+        val value = intGet(3)
+        val ref = refGet(2)
```

Comment:
This test does not actually test that `value` and `ref` actually work and produces the correct value.